data-prepper-plugins/aggregate-processor/src/test/java/com/amazon/dataprepper/plugins/processor/aggregate/AggregateProcessorTest.java

```diff
@@ -133,82 +134,99 @@ void setup() {
         when(pluginMetrics.timer(MetricNames.TIME_ELAPSED)).thenReturn(timeElapsed);
     }
 
-    private AggregateProcessor createObjectUnderTest() {
-        return new AggregateProcessor(aggregateProcessorConfig, pluginMetrics, pluginFactory, aggregateGroupManager, aggregateIdentificationKeysHasher, aggregateActionSynchronizerProvider);
-    }
-
     @Test
-    void handleEvent_returning_with_no_event_does_not_add_event_to_records_out() {
+    void getIdentificationKeys_should_return_configured_identification_keys() {
```

Comment:
This is the only new test, no need to review other tests. Had to move the existing tests to a @Nested class to avoid strict stubbing.